config/default.json

```diff
@@ -106,7 +106,11 @@
       "citizen-oauth-client-secret": "123456",
       "os-postcode-lookup-api-key": "AAAAAAA",
       "encryptionKey": "not-a-real-32-bit-encryption-key",
-      "AppInsightsInstrumentationKey":"APPINSIGHTS_INSTRUMENTATIONKEY"
+      "AppInsightsInstrumentationKey":"APPINSIGHTS_INSTRUMENTATIONKEY",
+      "launchDarkly-sdk-key": "sdk-AAAAA"
     }
+  },
+  "launchDarkly": {
+    "offline": true
```

Comment:
this should default to false, it can be set to true in `development.json` and / or `mocha.json` if you want it to be enabled for tests

Reply:
done